### PR TITLE
More Prop Fixes + Delete Broken Props

### PR DESCRIPTION
**Pull request by strubium**
## 🤔 What type of PR is this? (check all applicable)

- [ ] 🍕 Addition
- [ ] ⌨️ Productivity
- [ ] 🐛 Bug Fix
- [ ] 🔥 Optimization
- [ ] ⚙️ Configuration
- [x] 🌟 Quality Of Life
- [x] ✨ Enhancement
- [ ] 📝 Documentation

## 📝 Description

Removes broken/unusable props and updates bounding boxes for:

- Single Floodlight
- Shelf Top
- Server Rack Top

## 🧐 The Rationale

Props are going to be completely changed, but until then I will fix their hitboxes

## 🎯 Key Objectives

Do not break any props (I'm fixing the dumpster next pr)

## ⏮️ Backwards Compatibility 

1.12.2

## 📚 Related Issues & Documents

N/A

## 🖼️ Screenshots/Recordings

N/A

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [x] 🙅 no documentation needed

### CODE REVIEW

File: src/main/java/com/paneedah/mwc/tileentities/TileEntities.java [hunk 1]
```diff
@@ -1228,7 +1250,7 @@ public static void init(CommonProxy commonProxy) {
         new LootBoxConfiguration()
         .withMaterial(Material.IRON)
         .withName("ServerRackLeftAlt3")
-        .withModelClassName("com.paneedah.mwc.models.ServerRackLeftAlt3")
+        .withModelClassName("com.paneedah.mwc.models.ServerRackLeftAlt2")
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/tileentities/TileEntities.java:1253]: Why? It is named `Alt3` so why is it using `Alt2`

**Review by Desoroxxx — CHANGES_REQUESTED** [review]
Apart from that weird inconsistency, it's very nice

File: src/main/java/com/paneedah/mwc/tileentities/TileEntities.java (hunk 1 above)
Comment (strubium) [reply src/main/java/com/paneedah/mwc/tileentities/TileEntities.java:1253]: There is no icon for Alt3 so to make it different than the sandbag and stuff I made it use the Alt2 texture

Comment (Desoroxxx) [reply src/main/java/com/paneedah/mwc/tileentities/TileEntities.java:1253]: Yeah but `withModelClassName` will make it use the model of Alt2

Comment (strubium) [reply src/main/java/com/paneedah/mwc/tileentities/TileEntities.java:1253]: No, <code>.withTextureName</code> is the model (Dont ask me why idk)

I changed it when I fixed the office chairs (#193)

Comment (Desoroxxx) [reply src/main/java/com/paneedah/mwc/tileentities/TileEntities.java:1253]: Definitly not `.withTextureName` points towards the png of the texture as it should, and looked inside the code it is the model

Comment (Desoroxxx) [reply src/main/java/com/paneedah/mwc/tileentities/TileEntities.java:1253]: > No, `.withTextureName` is the model (Dont ask me why idk)
> 
> I changed it when I fixed the office chairs (#193)

Oh crums, well yeah the icons are tied to the model

Comment (strubium) [reply src/main/java/com/paneedah/mwc/tileentities/TileEntities.java:1253]: idk at this point. I'll change it back

Comment (strubium) [reply src/main/java/com/paneedah/mwc/tileentities/TileEntities.java]: Fixed

**Comment by strubium** [comment]
Fixed! Sorry

**Review by Desoroxxx — APPROVED** [review]

**Pull request merged**